packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```
@@ -128,6 +133,9 @@ exit 0
 %doc %{gem_instdir}/extra/dynflow-executor.example
 
 %changelog
+* Tue Feb 26 2019 Lukas Zapletal <lzap+rpm@redhat.com> 0.15-2
```

```suggestion
* Tue Feb 26 2019 Lukas Zapletal <lzap+rpm@redhat.com> 0.15.0-2
```

rpmlint is picky, and so am I ;-)

---

Oh thanks done.